src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnectorConfig.java

```diff
@@ -209,6 +218,17 @@ static ConfigDef newConfigDef() {
             5,
             ConfigDef.Width.NONE,
             SNOWFLAKE_SCHEMA)
+        .define(
+            SNOWFLAKE_ROLE,
+            Type.STRING,
+            null,
+            nonEmptyStringValidator,
+            Importance.LOW,
```

Comment:
Not in private preview though. Also, from code kafka code, I dont see any usage of Importance. Looking at the existing code, importance HIGH is for required values.  

Lets keep low for both the new configs since they are in preview mode. 

I will change it to HIGH once we declare GA. 

I created a JIRA for all such items: https://snowflakecomputing.atlassian.net/browse/SNOW-530333